br/backup-and-restore-faq.md

```diff
@@ -15,9 +15,25 @@ It is recommended to mount an NFS disk as a backup disk during backup. For detai
 
 ## How much does it affect the cluster during backup using BR?
 
-When you use the `oltp_read_only` scenario of `sysbench` to back up to a disk (make sure the backup disk and the service disk are different) at full rate, the cluster QPS is decreased by 15%-25%. The impact on the cluster depends on the table schema.
+For TiDB v5.3.0 or earlier versions, the default configuration parameters of BR only serve to **offline backup** tasks. In the default configuration, the backup tasks using BR might use a large amount of CPU and I/O resources of the cluster, which increases cluster latency.
 
-To reduce the impact on the cluster, you can use the `--ratelimit` parameter to limit the backup rate.
+The results of internal tests show that under the default configuration, the backup tasks using BR have a great impact on the cluster performance. The detailed test information is as follows:
+
+- Test environment: Use a cluster with 6 TiKV nodes and a single-table dataset to perform the following hybrid operations. The ratio of the read and write tasks is about `10:1`.
+- Test methods:
+    - Insert data to a single point
+    - Update data in a single point
+    - Queries for small ranges
+- Benchmark: Total QPS is around 20K. P95 latency is around 28ms. P99 latency is around 50ms.
```

Comment:
```suggestion
- Benchmark: Total QPS is around 20k. P95 latency is around 28 ms. P99 latency is around 50 ms.
```
Use a lowercase `k`. See [Using k to indicate thousands](https://developers.google.cn/style/units-of-measure#using-k-to-indicate-thousands).
Put a space before [the unit of measurement](https://developers.google.cn/style/units-of-measure).